lib/player.rb

```diff
@@ -63,6 +64,14 @@ def draw
     @font.draw_text("velocity: #{self.velocity}\ndirection: #{self.direction}", 0, 0, 0)
   end
 
+  def x_position
+    @x_position
+  end
+
+  def y_position
```

Comment:
you already created these methods by adding `attr_reader :x_position, :y_position` at the top